scripts/release-train-issue-template.md

```diff
@@ -43,9 +43,9 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 - [ ] Make sure there are no stray staging repos on sonatype
 - [ ] Wait until [master build finished](https://travis-ci.org/akka/akka-http/builds/) after merging the release notes (otherwise, the master
       build might pick up the tag and start publishing the release uhoh)
-- [ ] Create a tag for the release (e.g. `v13.3.7`) and push it.
-- [ ] Check that the Travis CI [release build](https://travis-ci.org/akka/akka-http/branches) has executed successfully
-- [ ] Go to https://bintray.com/akka/maven/akka-http and select the just released version
+- [ ] Create a tag for the release (e.g. `git tag -s -a v13.3.7 -m "Release 13.3.7"`) and push it.
```

Comment:
Can we splice in the concrete release here?

Reply:
:+1: 
```suggestion
- [ ] Create a tag for the release (e.g. `git tag -s -a v$VERSION$ -m "Release $VERSION$"`) and push it.
```